lib/tabular_form_builder.rb

```diff
@@ -55,6 +55,7 @@ def #{selector}(field, options = {}, *args)
 
   def label(method, text = nil, options = {}, &block)
     options[:class] = Array(options[:class]) + %w(form--label)
+    options[:title] = options[:title] || object.class.human_attribute_name(method)
```

Comment:
@floriank as the styles are applied to all labels, all labels should hence benefit from having a title. For that we need to add this to the `styled_label_tag` helper method as well.